Proper initialization of custom-theme-load-path.

PierreTechoueyres opened the pull request:
Add current directory to custom-theme-load-path in order to allow theme
listing with `customize-themes'.

PierreTechoueyres committed:
Proper initialization of custom-theme-load-path.

Add current directory to custom-theme-load-path in order to allow theme
listing with `customize-themes'.

jonnay commented:
Oh sweet!  This would totally fix the loading problem! 

I actually tangle the code from the org file.  would you mind making the addition there instead?
Thanks!

PierreTechoueyres committed:
Proper initialization of custom-theme-load-path.

Add current directory to custom-theme-load-path in order to allow theme
listing with `customize-themes'.

PierreTechoueyres commented:
Does it look like what you want ?

jonnay commented:
Perfect, thank you!
